test/unit/conftest.py

```diff
@@ -36,17 +36,21 @@ def autosubmit_exp(autosubmit: Autosubmit, request: pytest.FixtureRequest) -> Ca
     tmp_dir = TemporaryDirectory()
     tmp_path = Path(tmp_dir.name)
 
+
     def _create_autosubmit_exp(expid: str):
-        # directories used when searching for logs to cat
         root_dir = tmp_path
         BasicConfig.LOCAL_ROOT_DIR = str(root_dir)
-        exp_path = root_dir / expid
-        exp_tmp_dir = exp_path / BasicConfig.LOCAL_TMP_DIR
-        aslogs_dir = exp_tmp_dir / BasicConfig.LOCAL_ASLOG_DIR
-        status_dir = exp_path / 'status'
-        aslogs_dir.mkdir(parents=True, exist_ok=True)
-        status_dir.mkdir(parents=True, exist_ok=True)
-
+        exp_path = BasicConfig.expid_dir(expid)
+        
+        # directories used when searching for logs to cat
+        exp_tmp_dir = BasicConfig.expid_tmp_dir(expid) 
+        aslogs_dir = BasicConfig.expid_aslog_dir(expid) 
+        status_dir =exp_path / 'status'
```

Comment:
I'm changing everything to the `/` operator, but should be easy to change back if we decide to use `.joinpath`. We probably should discuss this and other similar choices with the entire team; for this particular one I'm not really aware of the difference (if there's any) between these two

Reply:
>I'm changing everything to the / operator, but should be easy to change back if we decide to use .joinpath. We probably should discuss this and other similar choices with the entire team;

Agreed!

> for this particular one I'm not really aware of the difference (if there's any) between these two

No difference :-)

https://github.com/python/cpython/blob/78ffba4221dcb2e39fd5db80c297d1777588bb59/Lib/pathlib/_local.py#L146-L158

They both call the same function in the end. The `/` operator was overloaded in `pathlib` for convenience. I still use `joinpath` when I need a function (like passing it to another function) for convenience. But otherwise I normally go with `/` as it reads equally simple and it's faster for me to type (except on the Spanish keyboard, as I never know where the keys are located :laughing: )

There are other operators used in Python too, like `+` with strings concatenation, or `|` `^` etc in sets (for union, difference, etc). I think that's also part of why they decided an operator was a good idea in pathlib (but it took me a while to get used to it).

https://github.com/python/cpython/blob/78ffba4221dcb2e39fd5db80c297d1777588bb59/Lib/pathlib/_local.py#L156